pages/opam-packaging.html

```diff
@@ -144,13 +171,17 @@ <h2>Creating a package definition file</h2>
 <code>md5=</code> are also supported.)
 </p>
 
-<p>Commit the new package:</p>
+<p>Create a new branch named <code>coq-foo.1.0.0</code>,
+and commit the new <code>opam</code> file:
+</p>
 
-<pre><code>git add released/packages/coq-foo/coq-foo.1.0.0
+<pre><code># Create the branch coq-foo.1.0.0 based on <code>upstream/master</code>
+git checkout -b coq-foo.1.0.0 upstream/master
```

Comment:
I'd recommend doing the branch creation step *before* working on adding the package definition, as it will avoid any conflict, in particular if the package has already been added by someone else in the archive, or if the branch you are in has much diverged from upstream/master.

Reply:
I see, that's a good point.